TheJumpCloud/support
# Sa 2825 sentinel one install scripts command gallery

## Issues
* [SA-2825](https://jumpcloud.atlassian.net/browse/sa-2825) - SA-2825-SentinelOne-Install-Scripts-Command-Gallery

## What does this solve?
Ability to install Sentinel One agent using Commands

## Is there anything particularly tricky?
* Linux and Windows S1 install does not work for arm based machines (M1 MacBooks)
* Make sure to uninstall the S1 agent after a successful install due to it that it can lock down your VM network and JC agent. [Uninstall instructions](https://www.cybervigilance.uk/post/sentinelone-uninstalling-agent)

## How should this be tested?

* Mac
1. Add the Sentinel One Agent Permissions Policy to your Mac Device
2. Create a .sh file, change the sentinelToken and DownloadURL variables and run this [script](https://github.com/TheJumpCloud/support/blob/SA-2825-SentinelOne-Install-Scripts-Command-Gallery/PowerShell/JumpCloud%20Commands%20Gallery/Mac%20Commands/Mac%20-%20Install%20Sentinel%20One%20Agent.md)
3. Sentinel One agent should pop up on the top right icons and it shouldn't ask for any security permissions
4. Uninstall using this command: sudo sentinelctl uninstall --local
- Paraphrase is located in the S1 console agent actions

* Windows
1. Edit siteToken and installerURL variables then run the [powershell script ](https://github.com/TheJumpCloud/support/blob/SA-2825-SentinelOne-Install-Scripts-Command-Gallery/PowerShell/JumpCloud%20Commands%20Gallery/Windows%20Commands/Windows%20-%20Install%20Sentine%20One%20Agent.md)
2. After successful install, uninstall the agent

* Linux
1. Edit token and DownloadURL then run the [script](https://github.com/TheJumpCloud/support/blob/SA-2825-SentinelOne-Install-Scripts-Command-Gallery/PowerShell/JumpCloud%20Commands%20Gallery/Linux%20Commands/Linux%20-%20Install%20Sentinel%20One.md) 
2. After successful install, uninstall the agent
